### PR TITLE
feat : new navbar

### PR DESCRIPTION
**Pull request by sudipkundu999**
navbar
Made this new navbar

Preview : https://deploy-preview-13--cobra-ui.netlify.app/docs/docs#Navigation

### CODE REVIEW

**Review by zoyatanweer — APPROVED** [review]
good and clean code. Good to merge

File: Components/navigation/navigation.css [hunk 1]
```diff
@@ -1,47 +1,53 @@
-.cui-navigation-container{
-    width: 100%;
-    height: 5rem;
-    display: flex;
-    align-items: center;
-    padding: 0 2rem;
+:root{
+    --primary:#334155;
+    --nav-hover:#29bae2;
+    --grey:rgb(167, 167, 167);
```
Comment (sumitk-singh) [inline Components/navigation/navigation.css]: from the point of consistency, either follow rgb or hex, better to convert rgb to hex

**Review by sumitk-singh — APPROVED** [review]
rest code looks clean. can merge with this small change.

File: Components/navigation/navigation.css (hunk 1 above)
Comment (sudipkundu999) [reply Components/navigation/navigation.css]: Thank you for pointing it out. Done!

**Pull request merged**